modules/salt/pages/custom-states.adoc

```diff
@@ -0,0 +1,151 @@
+  Do not use spaces.
+. In the [guimenu]``Description`` field, type a short description of the configuration your state performs.
+. In the [guimenu]``SLS Contents`` field, type the contents of your ``init.sls`` state.
+  Ensure your file starts by specifying the source of the managed file, using this syntax:
++
+----
+file.managed:
+  - source: salt://<org_name>/<channel_name>/etc/<filename>
+----
++
+ Example custom state files are given later in this section.
+. Click btn:[Update Channel] to save your state.
+
+
+
+.Procedure: Adding Additional Files to a Custom State Channel
+. In the {productname} {webui}, navigate to menu:Configuration[Channels].
+. Click the name of the channel you want to add files to.
+. To create a new file, click btn:``Create configuration file`` and type the contents of the file.
+. To upload an existing file, click btn:``Upload Configuration Files`` and select the file to upload.
+. To copy an existing file, click btn:``Import a File from Another Channel or System`` and select the file to copy.
+
+
+
+.Procedure: Editing a Custom Salt State
+. In the {productname} {webui}, navigate to menu:Configuration[Channels].
+. Click btn:[View/Edit <filename>.sls File].
+. Make your changes to the file
+. Click btn:[Update Configuration File] to save your state.
+
+You can also manage revisions, compare the state to others in your organization, and download the ``.sls`` file from this dialog.
+
+
+
+.Procedure: Assigning a Client to a Custom Salt State
+. In the {productname} {webui}, navigate to menu:Configuration[Channels].
+. Click the name of the state you want to assign a client to.
+. Navigate to the menu:Systems[Target Systems] tab.
+. Check the clients you want to assign.
+. Click btn:[Subscribe systems].
+
+
+
+== Example Custom State Files
+
+This section contains some example custom state files.
+Use these as a basis for writing your own custom states.
+
+
+
+.Example: Initial State File
+
+----
+file.managed:
+  - source: salt://example_org/example_channel/etc/motd
+[
+  FIXME: reference other file here somehow
+]
+----
+
+
+
+.Example: Custom State File
+
+----
+install required packages:
+  pkg.installed:
+    - pkgs:
+      - lsof
+    - failhard: True
+update packages:
+  module.run:
+    - name: pkg.upgrade
+    - failhard: True
+check if reboot is needed:
+  salt.set_reboot_needed:
+    - failhard: True
+reboot:
+   salt.reboot_if_needed:
+     - failhard: True
+----
+
+
+
+== Custom State to Trust a GPG Key
+
+By default, operating systems trust only their own GPG keys when they are installed, and do not trust keys provided by third party packages.
+The clients can be successfully bootstrapped without the GPG key being trusted.
+However, you cannot install new client tool packages or update them until the keys are trusted.
+
+Salt clients are set to trust {suse} tools channels GPG keys when they are bootstrapped.
+For all other clients and channels, you need to manually trust third party GPG keys.
```

Comment:
I would suggest from now on we talk about 3rd party channels and packages. We better use not examples with SLES or SUSE Manager keys prevent the assumption that other SUSE provided keys must be trusted on the clients.